SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestHandoverReplication/SpatialTestHandoverActorComponentReplication.h

```diff
@@ -26,14 +26,12 @@ namespace HandoverReplicationTestValues
 static constexpr int BasicTestPropertyValue = 0;
 static constexpr int UpdatedTestPropertyValue = 100;
 
-static constexpr float WorldSize = 1000.0f;
-
 // Positions that belong to specific server according to 1x2 Grid LBS.
 // Forward-Left, will be in Server 1's authority area.
-static const FVector Server1Position{ WorldSize / 4, -WorldSize / 4, 0.0f };
+static const FVector Server1Position{ 1000.0f, -1000.0f, 0.0f };
```

Comment:
Unsure about why this was removed.

Reply:
I switched to using the 2workerSmallInterest LB, so the world size is much greater and it did not seem like the test relied on the world size being small.

Reply:
Nah, it was just a convenience thing - keeping world size as constexpr lets us calculate cube positions in compile time rather than hardcoding them. Can we maybe create public constexpr values for world sizes, actually? 🤔

Reply:
Hmmm, usually I think the world size shouldn't matter. It's usually a large number just to make sure you don't leave the LB area.
In that sense, I think the essence of this const vector is that it is "some distance away from origin".